dev-resources/load-balanced-docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+# README:
+# Runs *two* SQL LRS containers next to each other behind a load balancer(nginx).
+
+# To run: 
+# 1.  Replace the main `lrsql/docker-compose.yml` file with this one
+# 2.  From the lrsql dir:  docker compose up
+# 3.  Access on port 8083 in your browser.
+# 4.  Comment in/out the `build` and `image` lines in lrs_1 and lrs_2 to switch between building from local source, or pulling the published lrsql image from the docker repository.
+
+
+# See the Docker Compose docs for more info: https://docs.docker.com/compose/
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: lrsql_user
+      POSTGRES_PASSWORD: lrsql_password
+      POSTGRES_DB: lrsql_db
+  lrs_1:
+    # build: . # switch to this for active dev
+    image: yetanalytics/lrsql:latest
+    command:
+      - /lrsql/bin/run_postgres.sh
+    ports:
+      - "8080:8080"
+    depends_on:
+      - db
+    environment:
+      LRSQL_API_KEY_DEFAULT: my_key
+      LRSQL_API_SECRET_DEFAULT: my_secret
+      LRSQL_ADMIN_USER_DEFAULT: my_username
+      LRSQL_ADMIN_PASS_DEFAULT: my_password
+      LRSQL_ALLOW_ALL_ORIGINS: true
+      LRSQL_DB_HOST: db
```

Comment:
couldn't help but notice this config will fail. `Error: missing sandwich`

Reply:
added sandwich